etc/profile-m-z/obsidian.profile

```diff
@@ -0,0 +1,88 @@
+# Firejail profile for obsidian
+# Description: Obsidian is the private and flexible writing app that adapts to the way you think.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include obsidian.local
+# Persistent global definitions
+include globals.local
+
+### Basic Blacklisting ###
+include disable-common.inc              # dangerous directories like ~/.ssh and ~/.gnupg
+include disable-devel.inc               # development tools such as gcc and gdb
+include disable-exec.inc                # non-executable directories such as /var, /tmp, and /home
+include disable-interpreters.inc        # perl, python, lua etc.
+include disable-programs.inc            # user configuration for programs such as firefox, vlc etc.
+include disable-xdg.inc                 # standard user directories: Documents, Pictures, Videos, Music
+
+### Home Directory Whitelisting ###
+whitelist ${HOME}/.gitconfig            # for the git plugin
+whitelist ${HOME}/.config/git           # for the git plugin
+whitelist ${HOME}/.pki/nssdb
+whitelist ${HOME}/.cache/AMD
+whitelist ${HOME}/.cache/nvidia
+whitelist ${HOME}/.local/share/vulkan
+whitelist ${HOME}/.local/share/vulkan/implicit_layer.d
+whitelist ${HOME}/.config/vulkan
+whitelist ${HOME}/.local/share/vulkan/loader_settings.d
+whitelist ${HOME}/.config/kdedefaults
+whitelist ${HOME}/.Xdefaults-desktop-pc
+whitelist ${HOME}/.config/kdedefaults/gtk-3.0
+whitelist ${HOME}/.cache/mesa_shader_cache
+whitelist ${HOME}/.local/share/applnk
+whitelist ${HOME}/.config/obsidian
```

Comment:
Add the following to disable-programs.inc:

```
blacklist ${HOME}/.config/obsidian
```

Reply:
Okay, I'll add that in the next commit.